arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -2968,15 +2968,44 @@ void RestReplicationHandler::handleCommandRevisionTree() {
     return;
   }
 
+  // shall we do a verification?
+  bool found{false};
+  bool withVerification{false};
+  std::string const& verificationString = _request->value("verification", found);
+  if (found) {
+    if (verificationString.compare("true") == 0) {
+      withVerification = true;
+    }
+  }
```

Comment:
Can be simplified to:
```suggestion
  bool withVerification = _request->parsedValue("verification", false);
```